clang_delta/CMakeLists.txt

```diff
@@ -57,16 +57,14 @@ include_directories(${CLANG_INCLUDE_DIRS}) # only works for LLVM post-4.0
 
 link_directories(${LLVM_LIBRARY_DIRS})
 
-llvm_map_components_to_libnames(LLVM_LIBS
-  coverage
-  irreader
-  mcparser
-  objcarcopts
-  option
-  passes
-  profiledata
-  support
-)
+find_program(LLVM_CONFIG_EXECUTABLE NAMES llvm-config DOC "llvm-config executable")
+if (NOT LLVM_CONFIG_EXECUTABLE STREQUAL "LLVM_CONFIG_EXECUTABLE-NOTFOUND")
```

Comment:
I think `NOT LLVM_CONFIG_EXECUTABLE` should be sufficient.

> False if the constant is 0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND.

https://cmake.org/cmake/help/v3.0/command/if.html